source/includes/api/_layer.md

```diff
@@ -1,180 +1,335 @@
+                    "center": {
+                        "lat": 39.6395375643667,
+                        "lng": -99.84375
+                    },
+                    "type": "dynamicMapLayer",
+                    "bbox": [
+                        -125.61,
+                        24.73,
+                        -66.81,
+                        49.07
+                    ]
+                },
+                "legendConfig": {
+                    "type": "choropleth",
+                    "items": [
+                        {
+                            "value": "< -90",
+                            "color": "#8C5107"
+                        },
+                        {
+                            "value": "-90 - -60",
+                            "color": "#BF822B"
+                        }
+                    ]
+                },
+                "interactionConfig": {},
+                "applicationConfig": {},
+                "staticImageConfig": {},
+                "createdAt": "2016-09-15T13:47:39.829Z",
+                "updatedAt": "2018-03-02T20:54:17.260Z"
+            }
+        }
+    ]
+}
+```
+
+This endpoint allows authenticated users to load all layers belonging to multiple datasets in a single request. 
+
+> Return all layers associated with two datasets, that are associated with either `rw` or `prep` applications 
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/layer?dataset=11de2bc1-368b-42ed-a207-aaff8ece752b
-curl -X GET https://api.resourcewatch.org/v1/dataset/11de2bc1-368b-42ed-a207-aaff8ece752b/layer
+curl -X POST "https://api.resourcewatch.org/layer/find-by-ids" \
+-H "Authorization: Bearer <your-token>" \
+-H 'Content-Type: application/json' \
+-d '{
+	"ids": ["<dataset 1 id>", "<dataset 2 id>"],
+    "app" "rw,prep"
+}'
 ```
 
-> Sort by name
+> Return all layers associated with two datasets, that are associated with both `rw` and `prep` applications simultaneously 
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/layer?sort=name
+curl -X POST "https://api.resourcewatch.org/layer/find-by-ids" \
+-H "Authorization: Bearer <your-token>" \
+-H 'Content-Type: application/json' \
+-d '{
+	"ids": ["<dataset 1 id>", "<dataset 2 id>"],
+    "app" "rw@prep"
+}'
 ```
 
-> Filter layers by status
+Besides the required `ids` array, your request body may optionally include a `app` string value if you'd like to filter the returned layers by their `application`:
+
+- Use a single value, like `rw`, if you want to show only layers that have `rw` as one of their applications. 
+- Use a comma separated list, like `rw,prep`, if you want to show only layers that have `rw` or `prep` as one of their applications.
+- Use a @ separated list, like `rw@prep`, if you want to show only layers that have both `rw` and `prep` as their applications.
+- None of the filters require exact matching - a layer that simultaneously contain the applications `rw`, `prep` and `gfw` would match all 3 filters above.
```

Comment:
This bullet doesn't make a ton of sense to me...

Reply:
what exactly is not clear? Should we work on the format, or on the message?

Reply:
I just didn't understand what the last bullet actually means. Now that I've read it a few times I get it. I will suggest some wording that might clear it up.

Reply:
```suggestion
- Note that the the filters do not need to match on the full application list. For example, the filters `rw,prep` and `re@prep` will both match a layer with the application list `["rw", "prep", "gfw"]`.
```